apps/cms.c

```diff
@@ -972,7 +970,7 @@ int cms_main(int argc, char **argv)
                 goto end;
 
             if (CMS_RecipientInfo_type(ri) == CMS_RECIPINFO_AGREE
-                && wrap_cipher) {
+                    && wrap_cipher != NULL) {
```

Comment:
Will leave it up to you whether this change stays in this PR.

Reply:
IMO it is OK to stay.

Reply:
yeah, it's a separate code-cleanup/style thing.   might as well leave it to avoid resetting the clock.  it would be done in #14182 anyway :)